INSTALL.md

```diff
@@ -2,12 +2,58 @@
 
 ## Linux for testing
 
-Install python 2.x (will probably work with 3.x with little modification)
+Install python 2.x (will probably work with 3.x with little modification).
 
-Install requirements:
+Install pip and other dependencies:
```

Comment:
You seem to have rebased the master branch on top of your branch. Can you get rid of these commits from your branch please.